ajax/libs/jquery.collapsible/package.json

```diff
@@ -13,10 +13,7 @@
     "div"
   ],
   "version": "1.2",
-  "license": {
-    "type": "BSD",
-    "url": "http://www.snyderplace.com/collapsible/license.txt"
-  },
+  "license": "BSD",
```

Comment:
It's actually `BSD-3-Clause`
> This license has also been called the "New BSD License" or "Modified BSD License"

See https://opensource.org/licenses/BSD-3-Clause for the explanation